src/games/behavmixed.imp

```diff
@@ -620,14 +567,13 @@ T MixedBehaviorProfile<T>::DiffNodeValue(const GameNode &p_node,
       // We've encountered the action; since we assume perfect recall,
       // we won't encounter it again, and the downtree value must
       // be the same.
-      return m_nodeValues(p_node->GetChild(p_oppAction->GetNumber())->GetNumber(),
-			  p_player->GetNumber());
+      return map_nodeValues[p_node->GetChild(p_oppAction->GetNumber())][p_player];
```

Comment:
There's now a version of `GetChild()` that just takes the action directly so we can avoid the `->GetNumber()`